test/packages/good/manifest.yml

```diff
@@ -1,4 +1,4 @@
-format_version: 1.0.4
+format_version: 2.0.0
```

Comment:
@elastic/ecosystem what should we do with test packages? I see some options:
1. Leave them as they are, update or add packages only to test new features in new versions.
1. Upgrade all of them to v2, copy `good` to `goodv1` and leave this one as reference for v1.
2. Duplicate all packages, leaving v1 packages to test backwards compatibility.

Reply:
I would go also for the option 1, to avoid having more folders/files copied.

I was wondering about the test packages whose goal is to test errors. Should they be updated/copied to test those errors in both formats v1 and v2 ?